datastore/simple/transformers/payload.go

```diff
@@ -100,12 +105,12 @@ func transformValue(value interface{}, includeTypes bool) (interface{}, bool) {
 	case *protos.Value_DoorValue:
 		outputType = "doorValue"
 		outputValue = map[string]bool{
-			"DriverFront":    v.DoorValue.DriverFront,
-			"PassengerFront": v.DoorValue.PassengerFront,
-			"DriverRear":     v.DoorValue.DriverRear,
-			"PassengerRear":  v.DoorValue.PassengerRear,
-			"TrunkFront":     v.DoorValue.TrunkFront,
-			"TrunkRear":      v.DoorValue.TrunkRear,
+			"FrontLeft":  v.DoorValue.FrontLeft,
```

Comment:
Is this going to be true for right hand drive countries too? This is a departure to how the Fleet API returns values.